_content/monographs/buddhism-in-thailand_karuna-kusalasaya.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Buddhism in Thailand: Its Past and Its Present"
+authors:
+  - " Karuna Kusalasaya"
+external_url: "https://www.buddhanet.net/pdf_file/bud-thailand.pdf"
```

Comment:
We almost always use the canonical BPS html for the external_url of a wheel:
https://www.bps.lk/olib/wh/wh085_Kusalasaya_Buddhistm-In-Thailand-Past-and-Present.html